include/battle.h

```diff
@@ -128,42 +128,40 @@ struct UnknownFlags
 
 struct DisableStruct
 {
-    /*0x00*/ u32 transformedMonPersonality;
-    /*0x04*/ u16 disabledMove;
-    /*0x06*/ u16 encoredMove;
-    /*0x08*/ u8 protectUses;
-    /*0x09*/ u8 stockpileCounter;
-    /*0x0A*/ u8 substituteHP;
-    /*0x0B*/ u8 disableTimer1:4;
-    /*0x0B*/ u8 disableTimer2:4;
-    /*0x0C*/ u8 encoredMovePos;
-    /*0x0D*/ u8 unkD;
-    /*0x0E*/ u8 encoreTimer1:4;
-    /*0x0E*/ u8 encoreTimer2:4;
-    /*0x0F*/ u8 perishSongTimer1:4;
-    /*0x0F*/ u8 perishSongTimer2:4;
-    /*0x10*/ u8 furyCutterCounter;
-    /*0x11*/ u8 rolloutCounter1:4;
-    /*0x11*/ u8 rolloutCounter2:4;
-    /*0x12*/ u8 chargeTimer1:4;
-    /*0x12*/ u8 chargeTimer2:4;
-    /*0x13*/ u8 tauntTimer1:4;
-    /*0x13*/ u8 tauntTimer2:4;
-    /*0x14*/ u8 battlerPreventingEscape;
-    /*0x15*/ u8 battlerWithSureHit;
-    /*0x16*/ u8 isFirstTurn;
-    /*0x17*/ u8 unk17;
-    /*0x18*/ u8 truantCounter:1;
-    /*0x18*/ u8 truantUnknownBit:1;
-    /*0x18*/ u8 unk18_a_2:2;
-    /*0x18*/ u8 unk18_b:4;
-    /*0x19*/ u8 rechargeCounter;
-    /*0x1A*/ u8 unk1A[2];
+     u32 transformedMonPersonality;
```

Comment:
where did the comment fields go? those are really helpful.

Reply:
They WERE helpful, now that all code that uses these structs has been decompiled, there is no need to have them.